README.md

```diff
@@ -110,17 +110,17 @@ For the purpose of setup, your project's root folder (where you can find your `.
 </p>
 
 ### From SPM
-The Amazon Chime SDK is available through [SPM](https://github.com/aws/amazon-chime-sdk-ios-spm). 
+The Amazon Chime SDK is available through [SPM](https://github.com/aws/amazon-chime-sdk-ios-spm). If you don't need video and content share functionality, or software video codec support, you could choose to use [no video codecs SPM](https://github.com/aws/amazon-chime-sdk-ios-no-video-codecs-spm) instead.
```

Comment:
nit: I'm not an grammar expert, feel free to push back😃: `If you don't need video, content share, or software video codecs support, you could choose [no video codecs SPM](https://github.com/aws/amazon-chime-sdk-ios-no-video-codecs-spm) instead.